eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/configuration/CommonConfiguration.java

```diff
@@ -15,10 +15,10 @@
 // * limitations under the License.
 // */
 //
-//package org.apache.runtime.configuration;
+//package com.webank.runtime.configuration;
 //
 //import com.google.common.base.Preconditions;
-//import org.apache.runtime.util.EventMeshUtil;
+//import com.webank.runtime.util.EventMeshUtil;
```

Comment:
here still com.webank

Reply:
Yes, I have noticed that. But I think there is no "org.apache.runtime" package...

Reply:
If the class is useless, we'd better remove it :D

Reply:
Ok, you can remove these classes later